includes/payment-methods/class-wc-gateway-stripe-p24.php

```diff
@@ -255,7 +255,7 @@ public function create_source( $order ) {
 
 		WC_Stripe_Logger::log( 'Info: Begin creating P24 source' );
 
-		return WC_Stripe_API::request( $post_data, 'sources' );
+		return WC_Stripe_API::request( apply_filters( 'wc_stripe_p24_source', $post_data, $order ), 'sources' );
```

Comment:
If instead adding new filter names, we do something like `apply_filters( 'wc_stripe_payment_source', $post_data, $order, 'p24' );` it will allow devs easily to do something like:
```
switch ( $source ) {
    case 'alipay': $post_data = ...
    case 'p24': $post_data = ...
}
```

But it's just a convenience so feel free to use what you have